java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S3752_java.json

```diff
@@ -19,6 +19,14 @@
   ],
   "defaultSeverity": "Blocker",
   "ruleSpecification": "RSPEC-3752",
-  "sqKey": "S3752",
-  "scope": "All"
+  "sqKey": "spring",
```

Comment:
this is weird change

Reply:
indeed this is strange, I don't know why this happened. I will revert for this PR